### PR TITLE
Add EOSC talk 

### PR DESCRIPTION
**Pull request by rantahar**
and remove timing from lightning talks section (this one is 10 minutes)

### CODE REVIEW

File: templates/schedule.html [hunk 1]
```diff
@@ -56,7 +56,7 @@ <h3>Tuesday, December 1st</h3>
         {{ macros::row_freeform(time="9:00", title="Welcome and Introduction to day’s schedule", color="light") }}
         {{ macros::row_list(time="9:05",
                             session="2.1",
-                            title="Lightning talks: Introducing groups (each 5 mins, slides are optional, max 3 slides)",
```
Comment (wleoncio) [inline templates/schedule.html:59]: Perhaps leaving "5 mins" as a suggestion so people have an idea of what "lightning" means here.

File: templates/schedule.html [hunk 2]
```diff
@@ -56,7 +56,7 @@ <h3>Tuesday, December 1st</h3>
...
+                            title="Introducing groups",
```
Comment (bast) [inline templates/schedule.html:59]:
```suggestion
                            title="Lightning talks: Introducing groups",
```

**Comment by bast** [comment]
I did not want to lose the word "lightning talks". But we can combine the lightning and talk session into one session if we don't want the distinction.

File: templates/schedule.html (hunk 1 above)
Comment (bast) [reply templates/schedule.html:59]: I agree, but this one talk is 10 minutes ...

**Comment by wleoncio** [comment]
Perhaps it can be classified as a "keynote lightning talk" or something.

**Pull request merged**